ui/README.md

```diff
@@ -12,6 +12,12 @@ Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 The page will reload if you make edits.<br />
 You will also see any lint errors in the console.
 
+#### API client & associated types
+
+Before running `yarn start`, make sure to run `sbt "backend/generateOpenAPISpec"` in the project's root directory. This command will generate the `<project_root>/backend/target/openapi.yaml` file.
```

Comment:
ok so I did `yarn start` after pulling these changes, and the terminal hangs on:

```
[~/projects/bootzooka/ui]% yarn start
yarn run v1.22.19
$ yarn generate:openapi-types && concurrently "react-scripts start" "yarn watch:openapi"
$ npx openapicmd typegen ../backend/target/openapi.yaml > src/api-client/openapi.d.ts
```

While in `src/api-client/openapi.d.ts` I got:

```
Need to install the following packages:
  openapicmd
Ok to proceed? (y) 
```

this probably should go to the terminal? ;)

Reply:
After answering "y" to the (hidden) question, I got:

```
/bin/sh: concurrently: command not found
error Command failed with exit code 127.
```

probably some setup that I'm missing?

Reply:
Ok so running `./frontend-start.sh`, which does `yarn install` helped ;) Maybe consolidating the READMEs would be a good idea?

Reply:
Now I got:

```
error @oclif/core@3.27.0: The engine "node" is incompatible with this module. Expected version ">=18.0.0". Got "16.15.1"
error Found incompatible module.
```

node 16 is specified as the minimum in `package.json`. Maybe this should be updated?